Src/Newtonsoft.Json-for-Unity.Tests/Assets/SystemWebPolyfill/HttpUtility.cs

```diff
@@ -0,0 +1,670 @@
+                            i += 5;
+                            continue;
+                        }
+                    } else if ((xchar = GetChar (bytes, i + 1, 2)) != -1) {
+                        acc.WriteByte ((byte) xchar);
+                        i += 2;
+                        continue;
+                    }
+                }
+
+                if (acc.Length > 0) {
+                    output.Append (GetChars (acc, e));
+                    acc.SetLength (0);
+                }
+
+                if (bytes [i] == '+') {
+                    output.Append (' ');
+                } else {
+                    output.Append ((char) bytes [i]);
+                }
+            }
+
+            if (acc.Length > 0) {
+                output.Append (GetChars (acc, e));
+            }
+
+            acc = null;
+            return output.ToString ();
+        }
+
+        public static byte [] UrlDecodeToBytes (byte [] bytes)
+        {
+            if (bytes == null)
+                return null;
+
+            return UrlDecodeToBytes (bytes, 0, bytes.Length);
+        }
+
+        public static byte [] UrlDecodeToBytes (string str)
+        {
+            return UrlDecodeToBytes (str, Encoding.UTF8);
+        }
+
+        public static byte [] UrlDecodeToBytes (string str, Encoding e)
+        {
+            if (str == null)
+                return null;
+
+            if (e == null)
+                throw new ArgumentNullException ("e");
+
+            return UrlDecodeToBytes (e.GetBytes (str));
+        }
+
+        public static byte [] UrlDecodeToBytes (byte [] bytes, int offset, int count)
+        {
+            if (bytes == null)
+                return null;
+            if (count == 0)
+                return new byte [0];
+
+            int len = bytes.Length;
+            if (offset < 0 || offset >= len)
+                throw new ArgumentOutOfRangeException("offset");
+
+            if (count < 0 || offset > len - count)
+                throw new ArgumentOutOfRangeException("count");
+
+            MemoryStream result = new MemoryStream ();
+            int end = offset + count;
+            for (int i = offset; i < end; i++){
+                char c = (char) bytes [i];
+                if (c == '+') {
+                    c = ' ';
+                } else if (c == '%' && i < end - 2) {
+                    int xchar = GetChar (bytes, i + 1, 2);
+                    if (xchar != -1) {
+                        c = (char) xchar;
+                        i += 2;
+                    }
+                }
+                result.WriteByte ((byte) c);
+            }
+
+            return result.ToArray ();
+        }
+
+        public static string UrlEncode(string str)
+        {
+            return UrlEncode(str, Encoding.UTF8);
+        }
+
+        public static string UrlEncode (string str, Encoding e)
+        {
+            if (str == null)
+                return null;
+
+            if (str == String.Empty)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4755183)